docs/servo_power_test/src/power_test_controller.ts

```diff
@@ -4,7 +4,7 @@ import {TestRunner} from './test_runner';
 import {Config} from './config';
 
 export type PowerData = [number, number];
-export type AnnotationData = [number, string];
+export type AnnotationDataList = {[labelName: string]: number};
```

Comment:
Maybe using ES6's Map instead of using Object is better.
c.f. https://google.github.io/styleguide/tsguide.html#indexable-key-string-type

Reply:
it seems that Map cannot be converted to JSON directly. https://typescriptbook.jp/reference/builtin-api/map#map%E3%81%AF%E7%9B%B4%E6%8E%A5json%E3%81%AB%E3%81%A7%E3%81%8D%E3%81%AA%E3%81%84
we have to create another variable, so it looks using extra memory. but is it okay...?

Reply:
It's not a memory-critical part and I think it would be better to take the benefits of type-safe (and less-magic-of-javascript) structure (Map).

Reply:
I got it!